testautoRIFT.py

```diff
@@ -218,11 +221,15 @@ def runAutorift(I1, I2, xGrid, yGrid, Dx0, Dy0, SRx0, SRy0, CSMINx0, CSMINy0, CS
     if CSMINx0 is not None:
         obj.ChipSizeMaxX = CSMAXx0
         obj.ChipSizeMinX = CSMINx0
-        chipsizex0 = float(str.split(subprocess.getoutput('fgrep "Smallest Allowable Chip Size in m:" testGeogrid.txt'))[-1])
-        try:
-            pixsizex = float(str.split(subprocess.getoutput('fgrep "Ground range pixel size:" testGeogrid.txt'))[-1])
-        except:
-            pixsizex = float(str.split(subprocess.getoutput('fgrep "X-direction pixel size:" testGeogrid.txt'))[-1])
+        if geogrid_run_info is None:
+            chipsizex0 = float(str.split(subprocess.getoutput('fgrep "Smallest Allowable Chip Size in m:" testGeogrid.txt'))[-1])
+            try:
+                pixsizex = float(str.split(subprocess.getoutput('fgrep "Ground range pixel size:" testGeogrid.txt'))[-1])
+            except:
+                pixsizex = float(str.split(subprocess.getoutput('fgrep "X-direction pixel size:" testGeogrid.txt'))[-1])
+        else:
+            chipsizex0 = geogrid_run_info['chipsizex0']
+            pixsizex = geogrid_run_info.get('pixsizex', geogrid_run_info['XPixelSize'])
```

Comment:
this could be simplified as `pixsizex = geogrid_run_info['XPixelSize']` as `pixsizex` should not be declared before here

Reply:
That matches the previous logic -- set `pixsizex` to the ground range pixel size for radar, as reported here:
    https://github.com/leiyangleon/autoRIFT/blob/master/geo_autoRIFT/geogrid/src/geogrid.cpp#L657
otherwise fall back to the X-direction pixel size.

This info should/will be set in the run-info block if the `FIXME`s in leiyangleon/Geogrid#7 are addressed. Specifically, this one:
    https://github.com/jhkennedy/Geogrid/blob/remote-access/testGeogrid_ISCE.py#L302

Are you saying this should *always* be the X-direction pixel size?

Reply:
Sounds good! Done.